charts/prometheus-statsd-exporter/templates/ingress.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "prometheus-statsd-exporter.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
+{{- if semverCompare ">=1.19.0-0" $kubeTargetVersion }}
+apiVersion: networking.k8s.io/v1
+{{- else }}
+apiVersion: networking.k8s.io/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "prometheus-statsd-exporter.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.ingress.tls }}
+  tls:
+    {{- range .Values.ingress.tls }}
+    - hosts:
+        {{- range .hosts }}
+        - {{ . | quote }}
+        {{- end }}
+      secretName: {{ .secretName }}
+    {{- end }}
+  {{- end }}
+  rules:
+    {{- range .Values.ingress.hosts }}
+    - host: {{ .host | quote }}
+      http:
+        paths:
+          {{- range .paths }}
+          - path: {{ . }}
+            {{- if semverCompare ">=1.19.0-0" $kubeTargetVersion }}
+            pathType: {{ .Values.ingress.pathType | default "Prefix" }}
```

Comment:
@scDisorder I'm not really familiar with `pathType` yet apart from reading the docs, but there it says `ImplementationSpecific` is the default. Should this default follow, or is there a reason we want `Prefix` here?

Reply:
wow, probably i misudnerstood the doc or took it at the wrong time
fixed